design/overview.md

```diff
@@ -80,19 +84,20 @@ be attributed to a signature from one of these sets, and proven **on the Cosmos
 a security level equal to the minimum of the Peg-Zone Validator Set, or reorganizing the Ethereum Chain 50 blocks.
 And provide a security equivalent to or greater than IBC.
 
-## Bootstraping
+## Bootstrapping
 
 We assume the act of upgrading the Cosmos-based binary to have peggy module is already complete,
 as approaches to that are discussed in many other places. Here we focus on the _activation_ step.
 
-1. Each `Operator` generates an Ethereum and Cosmos private key for the `Orchestrator`. These addresses are signed and submitted by the Operators valoper key. The `Orchestrator` is now free to use these delegated keys for all Peggy messages.
-1. Each `Operator` configures their `Orchestrator` with the `Peggy Bridge ID` and `start threshold` they wish, their `Orchestrator` is to reject any proposals that do not agree on these values.
-1. Anyone submits a Peggy contract using a known codehash and the current validator set of the Cosmos zone to an Ethereum compatible blockchain. Note that step #1 must happen first because the `Orchestrator` Ethereum Addresses must be made known in order for verification to be possible.
+1. Each `Operator` generates an Ethereum and Cosmos private key for the `Orchestrator`. These addresses are signed and submitted by the Operators valoper key in a MsgRegisterOrchestrator. The `Orchestrator` is now free to use these delegated keys for all Peggy messages.
+1. Governance votes are held individually on bridge parameters including `Peggy ID`, `Allowed validator set delta`, `start threshold`, and `Peggy contract code hash`
```

Comment:
I would not hold these individually.

There is a pre-defined ParamChangeProposal that can update one or more of them if they are stored together in one struct. It seems fine to allow the chain to vote once on an initial value set, then later vote to change just one. Declaring these as parameters voted by governance (or included in genesis) is plenty here I think. (The rest is implied with an SDK knowledge, but you can also be explicit of the above)